tests/qunit/setup/web-cli.js

```diff
@@ -3,4 +3,5 @@ const elementorWebCliConfig = {
 		rest: 'http://localtest/wp-json/',
 	},
 	nonce: 'test_nonce',
+	version: __karma__.config.qunit.elementorVersion,
```

Comment:
Check the lint issue

Reply:
https://github.com/elementor/elementor/pull/18421/commits/a7b8cd1256ff7f83d6e7d41becde3fbb688d510c